src/Cookie/ResponseCookie.php

```diff
@@ -91,12 +99,19 @@ public static function fromHeader(string $string) { /* : ?self */
                     case 'domain':
                         $meta = $meta->withDomain($pieces[1]);
                         break;
+
+                    default:
+                        $unknownAttributes[] = $part;
```

Comment:
All attributes are canonicalized (i.e. whitespaces around the `=` removed), just this one not.

Reply:
We simply won't change unrecognized attributes at all, I think that's fine.